plot_test.go

```diff
@@ -56,10 +56,10 @@ func TestLegendAlignment(t *testing.T) {
 		},
 		&recorder.Fill{
 			Path: vg.Path{
-				{Type: vg.MoveComp, X: 80, Y: 30},
-				{Type: vg.LineComp, X: 80, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 40},
-				{Type: vg.LineComp, X: 100, Y: 30},
+				{Type: vg.MoveComp, X: 80, Y: 31.865107548701296},
```

Comment:
While we are changing the magic constant, can you make it an actual `const` and use it instead of the literal when setting `Size` fields below (see line 121, for example)?